lemarche/cms/templatetags/advert_cms.py

```diff
@@ -0,0 +1,17 @@
+from django import template
+
+from lemarche.cms.snippets import Advert
+
+
+register = template.Library()
+
+
+# Advert snippets
+@register.inclusion_tag("includes/_advert_snippet.html", takes_context=True)
+def advert(context, layout="horizontal"):
```

Comment:
```suggestion
def advert_cms(context, layout="horizontal"):
```
actuellement la plupart de nos templatetags ont le même nom que le fichier, ca aide à s'y retrouver je pense, donc renommer à `advert_cms`, voir mieux : `cms_advert` :innocent: